integration-tests/hibernate-reactive-db2/pom.xml

```diff
@@ -97,12 +97,18 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <skip>true</skip>
+                    <systemProperties>
```

Comment:
@Sanne `systemProperties` is deprecated, `systemPropertyVariables` should be used instead. See also #9384 and #13481.

Reply:
thanks!  Will send a follow up soon.

Reply:
https://github.com/quarkusio/quarkus/pull/14107